### PR TITLE
Salary Manager Solution for non-profit organizations

### PR DESCRIPTION
**Pull request by duongnga**
#### 1Password Teams URL
https://upforshare.1password.com/

#### Project Name
Salary Manager Solution

#### Short Description
This is a free salary management and salary payment solution for non-profit organizations
#### Project Age
2 months on github and 8 months at local

#### Number Of Core Contributors
2
#### Project Website
https://github.com/duongnga/salary-manager-solution

#### Repository URL(s)
https://github.com/duongnga/salary-manager-solution

#### Latest Release URL(s)
https://github.com/duongnga/salary-manager-solution

#### Name
Duong Ngo

#### Email

duong@duongngo.com

#### Project Role
for non-profit organizations

#### Profile/Website
https://github.com/duongnga/salary-manager-solution

#### Can we contact you?
We'd love to be in touch to learn how you're using 1Password. Would you be open to our Developer Advocates reaching out?

Yes, I'm open.

### CODE REVIEW

**Comment by jodyheavener** [comment]
@duongnga Thank you for your application! It looks like you have inadvertently updated the README file to remove a handful of additional entries, and not added a description of your own project. If you could address this I will proceed with your application. Thank you.

**Comment by duongnga** [comment]
> @duongnga Thank you for your application! It looks like you have inadvertently updated the README file to remove a handful of additional entries, and not added a description of your own project. If you could address this I will proceed with your application. Thank you.

i'm fix it, check for me!

File: README.md [hunk 1]
```diff
@@ -3560,4 +3560,9 @@ https://github.com/pyiron
 ### LCMP-Minimal
 LCMP Minimal : Debian - Caddy - MariaDB - PHP
 
-https://github.com/bibicadotnet/LCMP-Minimal
\ No newline at end of file
+https://github.com/bibicadotnet/LCMP-Minimal
+
+### salary-manager-solution
+Salary Manage : Laravel - PHP - MariaDB
+
```
Comment (duongnga) [inline README.md]: This is a free salary management and salary payment solution for non-profit organizations

**Comment by jodyheavener** [comment]
Hi @duongnga - It looks like this problem persist. If you check [the diff tab](https://github.com/1Password/1password-teams-open-source/pull/902/files) it will show you that several entries are being removed. Are you able to give this another try? Thank you.

**Comment by jodyheavener** [comment]
Thanks for this, @duongnga! I've gone ahead and applied the promotion to your account.

If you haven't done so already, I would highly recommend implementing a [recovery plan for your team](https://support.1password.com/team-recovery-plan/) in case access for a particular contributor is ever lost. You are also welcome to add additional core contributors as you see fit.

Also, we'd love to hear more about your experience using 1Password in your development workflows! Feel free to join us over on the [1Password Developers Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-15k6lhima-GRb5Ga~fo7mjS9xPzDaF2A) workspace 🙂

**Comment by duongnga** [comment]
> Thanks for this, @duongnga! I've gone ahead and applied the promotion to your account.
> 
> If you haven't done so already, I would highly recommend implementing a [recovery plan for your team](https://support.1password.com/team-recovery-plan/) in case access for a particular contributor is ever lost. You are also welcome to add additional core contributors as you see fit.
> 
> Also, we'd love to hear more about your experience using 1Password in your development workflows! Feel free to join us over on the [1Password Developers Slack](https://join.slack.com/t/1password-devs/shared_invite/zt-15k6lhima-GRb5Ga~fo7mjS9xPzDaF2A) workspace 🙂

Thanks to @jodyheavener

**Pull request merged**